pg_analytics/tests/basic.rs

```diff
@@ -798,3 +798,23 @@ fn null_values(mut conn: PgConnection) {
     assert_eq!(rows[0].5, None);
     assert_eq!(rows[0].6, None);
 }
+
+#[rstest]
+fn copy_parquet_to_heap(mut conn: PgConnection) {
+    UserSessionLogsTable::setup().execute(&mut conn);
+    "CREATE TABLE heap AS SELECT * FROM user_session_logs".execute(&mut conn);
+
```

Comment:
Another way of doing this is with `SELECT INTO`:

```rust
    // "CREATE TABLE heap AS SELECT * FROM user_session_logs".execute(&mut conn);
    "SELECT * INTO heap FROM user_session_logs".execute(&mut conn);
```

That gives me an error when I try it:
```
"Error during planning: failed to resolve catalog: datafusion"
```

Reply:
This is a tough one...when I parse the `SELECT * INTO` statement into an AST it looks nearly identical to a normal SELECT statement. Best workaround I can think of is check for the `SELECT * INTO` string, but that feels a bit hacky.

Reply:
Yes indeed... you'd have to do a regex check for `SELECT .+ INTO` or something, because there could be any number of column identifiers in between. Seems like it would break easily.

Reply:
Good news I figured it out! We just needed to parse the logical plan instead of the AST.